src/applications/personalization/profile/tests/e2e/nametag.cypress.spec.js

```diff
@@ -52,6 +53,19 @@ describe('Profile NameTag', () => {
       nameTagRendersWithoutDisabilityRating();
     });
   });
+  context('when there is a 403 fetching the disability rating', () => {
```

Comment:
This is unrelated but I wanted to add the test mainly for documentation purposes.